CHANGELOG.md

```diff
@@ -8,9 +8,59 @@ and this project adheres to [Semantic Versioning].
 
 ## [0.9.0] — unreleased
 
+In this release we have been focused on improving performance of the Baseline interpreter.
+The end result is that the Baseline is **26% faster** than in previous version 0.8.0
+and **18% faster** than the Advanced interpreter while having 
+over **8x smaller** code analysis cost. The Baseline is now the _default_ interpreter because 
```

Comment:
I think the changes of the settings should be explained here as that is a "breaking" change.

Reply:
This is the first item in "Changed" section.